Sources_v3/StreamChatUI/ChatChannel/ChatMessageListVC.swift

```diff
@@ -301,6 +243,37 @@ open class ChatMessageListVC<ExtraData: ExtraDataTypes>: ViewController,
     }
 }
 
+// MARK: - ChatMessageActionsVCDelegate
+
+extension ChatMessageListVC: ChatMessageActionsVCDelegate {
```

Comment:
Do not put delegates in extensions, subclasses cannot override non-@objc methods from extensions.

Reply:
moved to class body in 4d8912880a8b7ba3079e3bb1719ff846440ebb24